Allocate a larger memory size for cmd

The length of the command line can be longer than 1024 sometimes,
which will cause a "Segmentation fault" error.

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

Upstream-Status: Pending
---
 tools/pkgdata/pkgdata.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/pkgdata/pkgdata.c b/tools/pkgdata/pkgdata.c
--- a/tools/pkgdata/pkgdata.c
+++ b/tools/pkgdata/pkgdata.c
@@ -439,7 +439,7 @@ main(int argc, char* argv[]) {
 /* POSIX - execute makefile */
 static int executeMakefile(const UPKGOptions *o)
 {
-    char cmd[1024];
+    char cmd[2048];
     /*char pwd[1024];*/
     const char *make;
     int rc;
-- 
1.7.10.2

